webapi/src/main/scala/org/knora/webapi/util/ActorUtil.scala

```diff
@@ -92,8 +92,8 @@ object ActorUtil {
       * @param message the message that was received.
       * @param log    a [[LoggingAdapter]].
       */
-    def handleUnexpectedMessage(sender: ActorRef, message: Any, log: LoggingAdapter)(implicit executionContext: ExecutionContext): Unit = {
-        val unexpectedMessageException = UnexpectedMessageException(s"Unexpected message $message of type ${message.getClass.getCanonicalName}")
+    def handleUnexpectedMessage(sender: ActorRef, message: Any, log: LoggingAdapter, who: String = "")(implicit executionContext: ExecutionContext): Unit = {
+        val unexpectedMessageException = UnexpectedMessageException(s"$who received an unexpected message $message of type ${message.getClass.getCanonicalName}")
```

Comment:
If `who == ""`, this error message won't make sense.

Instead, you could move this method to `ResponderV1`, and have it use `this.getClass.getName`.